geniza/corpus/admin.py

```diff
@@ -95,10 +95,9 @@ class DocumentAdmin(admin.ModelAdmin):
         'last_modified',
         'is_public'
     )
-    readonly_fields = ('old_input_by', 'old_input_date',
-                       'created', 'last_modified', 'shelfmark', 'id')
+    readonly_fields = ('created', 'last_modified', 'shelfmark', 'id')
     search_fields = ('fragments__shelfmark', 'tags__name', 'description',
-                     'old_input_by', 'notes', 'needs_review')
```

Comment:
I hope they don't still need to search on this! Maybe we could make a list filter if it's important (although they have a lot already)

Reply:
how would that work if we're removing the field?

Reply:
not on this field! a custom filter on users who have log entries for editing documents (but I hope we don't need this)

Reply:
(when I said "this" I meant the information, not the field! sorry)

Reply:
oh, makes sense! yeah, I can see how that'd work but let's not do it unless we have to.